src/components/characters/CharacterItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useThemePicker } from '../../hooks/themes';
+import '../page-theme/Theme.css';
+
+const CharacterItem = ({ name, image }) => {
+  const { theme } = useThemePicker();
+
+  return (
+    <figure className={theme}>
```

Comment:
I'm not sure this works as intended - you should use module css like you did in characterlist